client/my-sites/stats/summary/index.jsx

```diff
@@ -186,6 +187,15 @@ class StatsSummary extends Component {
 					statType="statsSearchTerms"
 					summary />;
 				break;
+
+			case 'anniversaries':
+				title = translate( 'Anniversaries' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **27** times:
`translate( 'Happy Anniversary!' )` *ES Score: 11*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).